railties/lib/rails/generators/rails/authentication/templates/app/controllers/concerns/authentication.rb.tt

```diff
@@ -14,6 +14,7 @@ module Authentication
 
   private
     def authenticated?
+      resume_session
```

Comment:
This will make `Current.session.present?` in the next line redundant

Reply:
Indeed... removing it.